### DISCUSSION TITLE
Disable printf in tinyusb?

### DISCUSSION THREAD

**Discussion by ObligedTester** [hathach/tinyusb#2332] [General]
Hello!

I have compiled a very small program on my RP2040 that is interfacing with a keyboard, the RP2040 being the host.

Just as a small concept, I wanted to know how much program space tinyusb is taking up, and how low its possible to get it.

I have completely disabled printf routines in the SDK and is just blinking a LED on keypress. However, looks like tinyusb is pulling it back in. I guess its for functionality for printing the "Panic" messages.

Is there a way in the config to disable this functionality?

**Comment by HiFiPhile** [comment]
`CFG_TUSB_DEBUG` must be defined somewhere, set it to 0 will disable all logs.

**Reply by ObligedTester** [reply]
Thank you!

That did it!
